python/python/bystro/cli/cli.py

```diff
@@ -328,10 +329,7 @@ def main():
 
     args = parser.parse_args()
     if hasattr(args, "func"):
-        try:
-            args.func(args)
-        except Exception as e:
-            print(f"\nSomething went wrong:\t{e}\n")
```

Comment:
Certain classes of error would not be reported under this scheme, like MemoryError for large requests.